app/views/layouts/_header.html.slim

```diff
@@ -5,41 +5,36 @@ section#header
-              - if current_user.try :supervisor?
-                li = link_to 'Dashboard', supervisor_dashboard_path
+      div.col-xs-offset-7.col-xs-5.col-sm-offset-1.col-sm-11.col-md-offset-3.col-md-9#menu-navbar
+        div class="navbar-collapse" id="menu"
+          ul class="nav navbar-nav navbar-right top navbar-default"
+            li = link_to 'Home &rarr;'.html_safe, 'http://railsgirlssummerofcode.org'
+            - if Season.projects_proposable?
+              li class="announcement"
+                = link_to 'Submit your Project', new_project_path
+            - if current_season.application_period?
+              li = link_to 'Projects', projects_path
+            - if show_application_link?
+              li = application_disambiguation_link
+            - if current_user.try :current_student?
               - if current_season.started?
-                li = link_to 'Activity',     root_path,  class: params[:controller] == 'activities' ? 'active' : ''
-              li = link_to 'Teams',        teams_path, class: params[:controller] == 'teams' ? 'active' : ''
-              li = link_to 'Community', users_path, class: params[:controller] == 'users' ? 'active' : ''
-              - if current_season.started?
-                li = link_to 'Conferences',  conferences_path,  class: params[:controller] == 'conferences' ? 'active' : ''
-              /- if signed_in? && current_user.roles.includes?('reviewer')  # TODO bring this into a reviewer dashboard
-              /  li = link_to 'Applications', applications_path
-              li = link_to 'Home &rarr;'.html_safe, 'http://railsgirlssummerofcode.org'
-    div.row
-      div.col-xs-offset-5.col-xs-7.col-md-offset-8.col-md-4
-        .text-right
-          nav.user
+                li = link_to 'Status Updates', [:students, :status_updates], class: params[:controller] == 'students/status_updates' ? 'active' : ''
+            - if current_user.try :supervisor?
+              li = link_to 'Dashboard', supervisor_dashboard_path
+            - if current_season.started?
+              li = link_to 'Activity',     root_path,  class: params[:controller] == 'activities' ? 'active' : ''
+            li = link_to 'Teams',        teams_path, class: params[:controller] == 'teams' ? 'active' : ''
+            li = link_to 'Community', users_path, class: params[:controller] == 'users' ? 'active' : ''
+            - if current_season.started?
+              li = link_to 'Conferences',  conferences_path,  class: params[:controller] == 'conferences' ? 'active' : ''
+            /- if signed_in? && current_user.roles.includes?('reviewer')  # TODO bring this into a reviewer dashboard
+            /  li = link_to 'Applications', applications_path
+            li class="divider"
             - if signed_in?
-              = "Signed in as #{link_to h(current_user.name), current_user} &middot; ".html_safe
```

Comment:
we still need to have a link to the current_user somewhere (maybe in the menu as well?) since it's the only place from which a user can see and edit their own profile. A link with text "My profile" or something would be enough I think.